espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/http/Router.java

```diff
@@ -169,16 +170,25 @@ public BaseResponse route(String uri, Method method, Map<String, String> params,
             System.out.println("Finished processing " + method + " request for '" + uri + "'");
             return appiumResponse;
         } catch (NoSuchElementException e) {
+            e.printStackTrace();
```

Comment:
the duplication here is a bit intense. could we create a method like:

```java
protected AppiumResponse errorResponse (e, AppiumStatus status) {
  e.printStackTrace();
  return new AppiumResponse<>(status, e.getMessage());
}
```

then in each of these catches we could just do `return errorResponse(e, AppiumStatus.NO_SUCH_ELEMENT);`

?